composeApp/src/commonMain/kotlin/org/ooni/engine/models/TaskLogLevel.kt

```diff
@@ -0,0 +1,31 @@
+package org.ooni.engine.models
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+
+@Serializable(with = TaskLogLevelSerializer::class)
+enum class TaskLogLevel(val value: String) {
+    Debug("DEBUG2"),
+    Info("INFO"),
+}
+
+object TaskLogLevelSerializer : KSerializer<TaskLogLevel> {
+    override val descriptor =
+        PrimitiveSerialDescriptor("TaskLogLevel", PrimitiveKind.STRING)
```

Comment:
Do we want to use class name here?

Reply:
The official examples use the class name of the type we're serializing: https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/serializers.md#primitive-serializer